CHANGELOG.md

```diff
@@ -1,6 +1,28 @@
 ## Unreleased
 
-* [FEATURE] Add `-kubelet-endpointslice` argument to support `EndpointSlice` for the kubelet controller.
+## 0.77.0 / 2024-09-18
+
+> Along with this release we also introduce a new command line tool [poctl](https://github.com/prometheus-operator/poctl). It is designed specifically for managing Prometheus Operator Custom Resources.
+> This project is currently under active development and fully experimental, so expect breaking changes and rough edges. We encourage you to [try it out](https://github.com/prometheus-operator/poctl/releases/tag/v0.1.0) and provide the feedback.
+
+* [CHANGE] Add more API validation and changes in ScrapeConfig CRD. #6921 #6855
+* [CHANGE] Update API validations for ECSDConfig in ScrapeConfig CRD. #6744
+* [FEATURE] Add experimental Prometheus 3 support. This will allow users to try new Prometheus `v3.0.0-beta.0`. #6940
+  * Feature requests in this version won't be supported till Prometheus 3 is stable but this would let users to try the beta version with Operator and report issues if any.
+* [FEATURE] Allow Prometheus Agent to update, delete, and handle changes to DaemonSet object. #6787
+* [FEATURE] Add support PodMonitor for Prometheus Agent in DaemonSet mode. #6879
+* [FEATURE] Add `-kubelet-endpointslice` argument to support `EndpointSlice` for the kubelet controller. #6882
+* [FEATURE] Reject PrometheusRule objects that generate > 512KiB configmap. #6606
+* [FEATURE] Add support for `OutOfOrderTimeWindow` field for PrometheusAgent. #6851
+* [FEATURE] Add support for `MatchFirstNetwork` field for DockerSD. #6849 #6894
+* [FEATURE] Add support for new otlp receiver configuration. #6780
+* [FEATURE] Add support for file paths for certificate, key and client CA in web TLS. #6358
+* [FEATURE] Add new proxy fields into the AlertmanagerConfig CRD for the OAuth2 clients. #6922
+* [FEATURE] Add ProxyConfig fields into the AlertmanagerConfig CRD for the HTTP clients. #6864
+* [FEATURE] Add Ionos Service Discovery support to ScrapeConfig CRD. (#6732)
+* [BUGFIX] Fix AlertmanagerConfig objects with invalid child routes definition to cause problems. #6931
```

Comment:
I added it to the CHANGELOG.md already but at a wrong place :facepalm: 
https://github.com/prometheus-operator/prometheus-operator/blob/c846674a17c6db9d1bd04d08300c354f960d909b/CHANGELOG.md?plain=1#L11